Sources/Support/PaywallExtensions.swift

```diff
@@ -48,7 +51,10 @@ extension StoreView where Icon == EmptyView, PlaceholderIcon == EmptyView {
 
     /// Creates a view to load a collection of products from the App Store, and merchandise them.
     /// When the user purchases products through this paywall, the `RevenueCat` SDK will handle
-    /// the result automatically. All you need to do is to dismiss the paywall
+    /// the result automatically. All you need to do is to dismiss the paywall.
+    ///
+    /// - Warning: In order to use StoreKit paywalls you must configure the `RevenueCat` SDK
```

Comment:
I haven't been tracking the Paywall changes, but would it make sense to log an error if a user tries to display the paywall using SK1? I was also thinking of actually erroring out, but not sure how that would work, so a log might be enough.

Reply:
That's done already :) https://github.com/RevenueCat/purchases-ios/blob/4.25.6/Sources/Support/PaywallExtensions.swift#L112